tests/scripts/scripts_path.py

```diff
@@ -15,3 +15,6 @@
 sys.path.append(os.path.join(os.path.dirname(__file__),
                              os.path.pardir, os.path.pardir,
                              'scripts'))
+sys.path.append(os.path.join(os.path.dirname(__file__),
```

Comment:
Isn't it the case that we should rather remove this file and change files that import scripts_path to rather import framework_path?

Reply:
In practice I believe we need both files for 2 reasons:

1. Each file needs to be in the same directory as the scripts that import it, so we need one for `scripts/` and one for `tests/scripts`.
2. Our test scripts still need to import modules from `scripts` that aren't in `mbedtls_dev`, so have not been moved to the framework. For example, `tests/scripts/depends.py` imports `scripts/config.py`.

Reply:
Yes that makes sense. Thanks for writing this down.